tox.ini

```diff
@@ -1,15 +1,16 @@
-
 [tox]
 minversion = 1.8
 envlist =
-    py{27}-ansible{24,25,26}-functional
-    py{27,36,37}-ansible{24,25,26}-unit
-    py{36,37}-ansible{24,25,26}-functional
+    py{27,36,37}-ansible{24,25,26}-functional
```

Comment:
Please wait until I extract this from my PR. I've got it extended with a few more.